### PR TITLE
Add gh workflows from PLAYGROUND

### PR DESCRIPTION
**Pull request by jgara**
Issues:
- Integration tests (`mvn verify`) do not work. Commented out in `ci.yml`

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  call-workaround:
+    name: Run Parent POM Workaround
+    uses: eclipse-pass/pass-grant-loader/.github/workflows/parent-pom-workaround.yml@main
+  
+  call-tests-unit:
+    name: Run Unit Tests
+    uses: eclipse-pass/pass-grant-loader/.github/workflows/tests-unit.yml@main
+    needs: call-workaround
+    
+#  call-tests-integration:
```
Comment (aforward) [inline .github/workflows/ci.yml:24]: Consider a comment to outline the underlying issue for why these are breaking (and not yet part of the CI).

**Review by aforward — COMMENTED** [review]
One comment.

File: .github/workflows/ci.yml (hunk 1 above)
Comment (jgara) [reply .github/workflows/ci.yml:24]: Can confirm this is related to: https://github.com/eclipse-pass/main/issues/250. For example,
```
[ERROR] processGrantIT(org.dataconservancy.pass.grant.integration.JhuPassUpdaterIT)  Time elapsed: 0.053 s  <<< ERROR!
java.lang.RuntimeException: A problem occurred while attempting to create a Resource: Failed to update null - unexpected status code 400: Could not parse jsonld: loading remote context failed: https://oa-pass.github.io/pass-data-model/src/main/resources/context-3.5.jsonld
```

File: .github/workflows/tests-integration.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
+name: Run Unit Tests
```
Comment (aforward) [inline .github/workflows/tests-integration.yml:1]: `Run Integration Tests`

**Review by aforward — CHANGES_REQUESTED** [review]
Small text change

**Review by aforward — APPROVED** [review]

**Pull request merged**